scalafmt-tests/src/test/resources/trailing-commas/trailingCommasPreserve.stat

```diff
@@ -63,6 +162,61 @@ def a(b: Int, c: Int,
  /* comment */ )
 >>>
 def a( b: Int, c: Int /* comment */ )
+<<< keep, do not fold, with spaces and comment after break
+spaces.inParentheses = true
+maxColumn = 40
+danglingParentheses.preset = false
+rewrite.trailingCommas.allowFolding = false
+===
+def a(b: Int, c: Int,
+ /* comment */ )
+>>>
+def a(
+    b: Int,
+    c: Int,
+    /* comment */
+)
+<<< keep, do not fold, with spaces and comment after break, binpack
+spaces.inParentheses = true
+maxColumn = 40
+binPack.preset = true
+importSelectors = binPack
+danglingParentheses.preset = false
+rewrite.trailingCommas.allowFolding = false
+===
+def a(b: Int, c: Int,
+ /* comment */ )
+>>>
+def a(
+    b: Int, c: Int, /* comment */
+)
+<<< keep, do not fold, with spaces and comment before break
+spaces.inParentheses = true
+maxColumn = 40
+danglingParentheses.preset = false
+rewrite.trailingCommas.allowFolding = false
+===
+def a(b: Int, c: Int, /* comment */
+)
+>>>
+def a(
+    b: Int,
+    c: Int, /* comment */
+)
+<<< keep, do not fold, with spaces and comment before break, binpack
+spaces.inParentheses = true
+maxColumn = 40
+binPack.preset = true
+importSelectors = binPack
```

Comment:
```suggestion
```

This is probably not neccessary.

Reply:
you're right, there are no imports but added anyway for consistency, in case the test is amended in the future, as the title of the test says "binpack".